FluentTc/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Text;
+
+namespace FluentTc.Extensions
+{
+    public static class StringExtensions
+    {
+        public static string FromPascalToCapitalizedCase(this string value)
+        {
+            StringBuilder sb = new StringBuilder();
```

Comment:
Maybe it's better to initialize StringBuilder withe the size of the original string in order to prevent allocationsL
```C#
new StringBuilder(value.Length)
```